cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSink.java

```diff
@@ -624,7 +632,7 @@ private void persistDistanceEvent(NGSIEvent event) throws Exception {
                     LOGGER.info("[" + this.getName() + "] Persisting data at NGSICartoDBSink. Schema (" + schema
                             + "), Table (" + tableName + "), Data (" + rows + ")");
                     backends.get(schema).insert(schema, tableName, withs, fields, rows);
-                } // try catch
+                } // try catch // try catch
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1286/commits/e27136711d99f5e76d739beba510ff7f13b9749f